src/main/flight/mixer.c

```diff
@@ -502,16 +504,21 @@ static void applyMixerAdjustmentLinear(float *motorMix, const bool airmodeEnable
     throttle = constrainf(throttle, -minMotor, 1.0f - maxMotor);
 }
 
-static float calcEzLandLimit(float maxDeflection)
+static float calcEzLandLimit(float maxDeflection, float speed)
 {
     // calculate limit to where the mixer can raise the throttle based on RPY stick deflection
     // 0.0 = no increas allowed, 1.0 = 100% increase allowed
-    float ezLandLimit = 1.0f;
-    if (maxDeflection < mixerRuntime.ezLandingThreshold) { // roll, pitch and yaw sticks under threshold
-        ezLandLimit = maxDeflection / mixerRuntime.ezLandingThreshold; // normalised 0 - 1
-        ezLandLimit = fmaxf(ezLandLimit, mixerRuntime.ezLandingLimit); // stay above the minimum
-    }
-    return ezLandLimit;
+    const float deflectionLimit = fminf(1.0f, maxDeflection / mixerRuntime.ezLandingThreshold);
```

Comment:
```suggestion
    const float deflectionLimit =  mixerRuntime.ezLandingThreshold > 0 ? fminf(1.0f, maxDeflection / mixerRuntime.ezLandingThreshold) : 0.0f;
```

Reply:
I was sure ez_landing_threshold was limited to 1, exactly to avoid this possible division by 0. it's not. Nice catch.

Reply:
should be fixed now